config/crd/bases/mcad.ibm.com_appwrappers.yaml

```diff
@@ -53,22 +53,10 @@ spec:
                             resource type
                           format: int32
                           type: integer
-                        apiVersion:
-                          description: 'APIVersion defines the versioned schema of
-                            this representation of an object. Servers should convert
-                            recognized schemas to the latest internal value, and may
-                            reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
-                          type: string
                         completionstatus:
-                          description: Optional field that drives completion status of appwrapper.
-                            This field within an item of an appwrapper determines the full state of appwrapper.
-                            The completionstatus field contains a list of conditions that make the associate item considered 
-                            completed, for instance :- completion conditions could be "Complete" or "Failed".
-                            The associated item's level .status.conditions[].type field is monitored for any one of these conditions.  Once all items with this 
-                            option is set and the conditionstatus is met the entire appwrapper state will be changed to one of the valid appwrapper completion state. Note :- this is an AND 
-                            operation for all items where this option is set.  See the list of appwrapper states for a list of valid complete states.
+                          description: Optional field for users to determine completion
```

Comment:
I think is best to update the descriptions from the app wrapper back into `golang` struct. We'll be loosing a lot of information otherwise. For better or worse the CRD(s) yawls were the "system of record" for documentation.